test/wh_test_crypto.c

```diff
@@ -253,6 +253,9 @@ int whTest_CryptoClientConfig(whClientConfig* config)
         WH_ERROR_PRINT("CURVE25519 shared secrets don't match\n");
     }
 
+    /* Tell server to close */
+    wh_Client_CommClose(client);
```

Comment:
Should this be called inside of wh_Client_Cleanup? Users should be able to call it manually to reset just the connection but it seems convenient to also add it to cleanup

Reply:
This is an async request that we are just busy spinning waiting for it to complete.  wh_Client_Cleanup is expected to synchronously clean up local resources, which could also have the transport zero-out/cancel any requests.  So, this must be outside of the cleanup call.